doc/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
+
+Date: 2023-06-27
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decisions made on this project.
```

Comment:
Can you expand on the why here? You can get it directly from https://github.com/asyncapi/studio/issues/663 if you don't want to scratch your head 😄